From 10d03bbfbc30a4ff596ffc77b73acdceef5a7826 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Thu, 17 Sep 2015 11:33:40 -0400
Subject: [PATCH] dir-locals: Avoid use of setq which triggers Emacs warning

Emacs has a whitelist of "safe" variables, using `setq` overrides
that and causes it to warn when opening any file by default.

Dropping the `setq` makes Emacs do the right thing.
---
 .dir-locals.el | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/.dir-locals.el b/.dir-locals.el
index cbbcec4..8130058 100644
--- a/.dir-locals.el
+++ b/.dir-locals.el
@@ -1,3 +1,3 @@
-((nil . ((setq sh-basic-offset: 4)
-         (setq indent-tabs-mode nil)
+((nil . ((sh-basic-offset . 4)
+         (indent-tabs-mode . nil)
          )))
